### PR TITLE
534 logging

### PR DESCRIPTION
**Pull request by sanjuroj**

### CODE REVIEW

File: config/sample-settings.secret.yml [hunk 1]
```diff
@@ -82,4 +82,4 @@ EMAIL_HOST_PASSWORD: 'XXXXXXXXXXXX'
 GOOGLE_ANALYTICS_PROPERTY_ID: 'UA-1111111-11'
 GOOGLE_ANALYTICS_DOMAIN: 'example.org'
 
-
+LOGFILE: '/path/to/your/log/file.log'
```
Comment (kurtwall) [inline config/sample-settings.secret.yml]: Maybe drop a comment above this line indicating the default if this isn't specified?

Comment (axoplasm) [reply config/sample-settings.secret.yml]: +1 to Kurt's comment

**Comment by sanjuroj** [comment]
@axoplasm, @kurtwall How about now?  I put the default back in.

File: tola/settings/base.py [hunk 2]
```diff
@@ -398,3 +343,29 @@
...
+    'formatters': {
+        'standard': {
+            'format': '%(asctime)s %(levelname)-8s %(name)-12s %(message)s',
+        }
+    },
+    'handlers': {
+        'file': {
+            'level': 'ERROR',
+            'class': 'logging.FileHandler',
+            'filename': 'error.log',
```
[the remark this answers is not in the document]
Comment (sanjuroj) [reply tola/settings/base.py]: Good idea, updated the PR.

**Pull request merged**